<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SVSimulationNetworkOptions</class>
 <widget class="QWidget" name="SVSimulationNetworkOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>935</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Network to simulate</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBoxNetwork"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Type of simulation</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboBoxModelType"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Default/Initial fluid temperature [C]</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QtExt::ValidatingLineEdit" name="lineEditDefaultFluidTemperature"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Reference Pressure [Pa]</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QtExt::ValidatingLineEdit" name="lineEditReferencePressure">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtExt::ValidatingLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">QtExt_ValidatingLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
